cmd/crc/cmd/start.go

```diff
@@ -199,3 +195,44 @@ func checkIfNewVersionAvailable(noUpdateCheck bool) {
 	}
 	logging.Debugf("No new version available. The latest version is %s", newVersion)
 }
+
+const startTemplate = `Started the OpenShift cluster.
+
+The server is accessible via web console at:
+  {{ .ClusterConfig.WebConsoleURL }}
+
+Log in as administrator:
+  Username: {{ .ClusterConfig.AdminCredentials.Username }}
+  Password: {{ .ClusterConfig.AdminCredentials.Password }}
+
+Log in as user:
+  Username: {{ .ClusterConfig.DeveloperCredentials.Username }}
+  Password: {{ .ClusterConfig.DeveloperCredentials.Password }}
+
+Use directly the 'oc' command line:
```

Comment:
This is worded incorrectly. My preference would be "Use the `oc` command line interface".

Reply:
done thx